lib/player.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Player
+  def initialize(marker)
+    @marker = marker
+    @board_index = 0
+  end
+
+  def select_position
+    puts 'select number between 1 to 9'
```

Comment:
Same output issue as above, this one would take a little more refactoring though.